trade-finance-manager-api/src/v1/__mocks__/amendmentVariables.js

```diff
@@ -226,10 +226,17 @@ const dealWithNoCompanyName = {
     bankInternalRefName: 'UKEF',
   },
 };
-
-const dealWithNoId = {
+const dealWithNoMongoId = {
```

Comment:
To be explicit on which Id as `dealWithNoId` can be confused for either MongoId or UKEF Deal ID.